idl2library.cmake

```
@@ -185,6 +182,7 @@ function(idl2library)
             ${${current_idl_target}_RELPATH}
             OPENDDS_IDL_OPTIONS ${current_idl_include_opts}
             TAO_IDL_OPTIONS ${current_idl_include_opts}
+            INCLUDE_BASE ${${current_idl_target}_ABSDIR}
```

> I think I understand what you are saying is the problem. But what do you propose we do to solve it? Are you going to make another opendds change to fix issue 2?

No, 2 is a decision that has to be made in OpenDDW here:

```suggestion
            INCLUDE_BASE "${CMAKE_CURRENT_SOURCE_DIR}"
```

That's the change that was needed for my example, but what this needs to be depends on how the IDL is organized. My IDL was in different directories because that's how I test `INCLUDE_BASE`, but `INCLUDE_BASE ${${current_idl_target}_ABSDIR}` would only work if all the IDL is all in the same directory. I guess what really matters is does `INCLUDE_BASE ${${current_idl_target}_ABSDIR}` work for all your IDL? If so then how this is now is probably okay, and we can merge this.

---

OK.  I think I understand better.  Lets look at this from a bit different angle:  The goal of this PR is to have it work the same as it did before without the warning messages.  Have we achieved that?  We can look into making it better in the future.

---

It will probably work the same, but that really depends on the IDL. Have you tested with all your IDL?

---

Yes.  I have tested it with both our project and our examples.